src/smdba/postgresqlgate.py

```diff
@@ -201,7 +205,7 @@ def check(self) -> bool:
         if int(pg_version[0]) < minversion[0] or (int(pg_version[0]) == minversion[0] and int(pg_version[1]) < minversion[1]):
             raise GateException("Core component is too old version.")
 
-        if not os.path.exists("/etc/sysconfig/postgresql"):
+        if not os.path.exists(self.config_file):
```

Comment:
If you want to do it in this PR you could already move this check to `__init__` as we discussed in #48 . Or we do it in a follow-up PR, either way works for me :)

Reply:
ok, added.